src/main/java/net/sf/marineapi/nmea/io/UDPDataReader.java

```diff
@@ -49,25 +49,36 @@
 	}
 
 	@Override
+	/**
+	 * Cleaned up and commented - no functional change.
+     * @author Bob Schwarz

```

Comment:
Author tags should be used only for packages, classes and interfaces.

Reply:
Done